<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>9</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="BarsTab">
     <attribute name="title">
      <string>Bars Chart</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="HorizontalStackedBarsTab">
     <attribute name="title">
      <string>H-Bars Chart</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="LineTab">
     <attribute name="title">
      <string>Line Chart</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="PieTab">
     <attribute name="title">
      <string>Pie Chart</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
